src/asset_util/src/lib.rs

```diff
@@ -191,16 +235,21 @@ lazy_static! {
 
 /// Takes a list of assets and returns a [CertifiedAssets] struct containing the assets and their
```

Comment:
nit: this line could be also improved, as "Takes a list of assets" is not really informative (as one sees it immediately).